PowerUp/app/src/main/res/values/strings.xml

```diff
@@ -17,5 +17,30 @@
     <string name="enter">Continue..</string>
     <string name="back_to_map">Back to Map</string>
     <string name="replay">Replay</string>
+    <string name="about_text"> <b><u>Welcome to PowerUp!</u></b> \n
```

Comment:
Don't use html code inside the string. Instead, create a layout with several elements (possibly several textViews and maybe images) to create the structure you want: Bold text or darker than other, text on different lines, etc.

I would recommend against using underlined text because of aesthetic reasons